tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -231,7 +247,11 @@
       {
         "name": "envoy_metrics_service",
         "type": "STRICT_DNS",
+{{- if .ipv6_proxy }}
```

Comment:
I would also pass this as an explicit param - {{ .DnsLookupFamily }} and set it in code.
Same reason - avoiding complex templates.

Reply:
`{{ .DnsLookupFamily }}` can be defined but where the value would come from? I mean the process rendering this template how it would chose which value to assign this variable?